### PR TITLE
v6.2.0

### PR DESCRIPTION
**Pull request by ashleynolan**
### Changed
- Updated the CSS task so that it produces map files for both the concatenated and the minified files produced

### CODE REVIEW

File: tasks/css.js [hunk 1]
```diff
@@ -72,7 +74,7 @@ gulp.task('scss:lint', () => gulp.src([`${pathBuilder.scssSrcDir}/**/*.scss`, ..
  * Uses our config rules to validate syntax and structure of the CSS.
  *
  */
-gulp.task('css:lint', () => gulp.src(`${pathBuilder.cssDistDir}/**/*.css`)
+gulp.task('css:lint', () => gulp.src([`${pathBuilder.cssDistDir}/**/*.css`, `-${pathBuilder.cssDistDir}/**/*.css.map)`])
```
Comment (DamianMullins) [inline tasks/css.js]: What does the addition to this line do? Exclude the map file?

Comment (ashleynolan) [reply tasks/css.js]: Yeh – it seems to include the map files as part of the stream without this and tries to minify them…

Comment (DamianMullins) [reply tasks/css.js]: Are you sure the minus operator works? The `!` is usually the operator to use to exclude files?

Comment (DamianMullins) [reply tasks/css.js]: I've just tested with this addition removed and the output was the same?

Comment (ashleynolan) [reply tasks/css.js]: Just tested and for some reason, it no longer errors.  Go figure (I swear it didn’t work the other day…!).

Have pushed the amend.

File: tasks/css.js [hunk 2]
```diff
@@ -72,7 +74,7 @@ gulp.task('scss:lint', () => gulp.src([`${pathBuilder.scssSrcDir}/**/*.scss`, ..
  * Uses our config rules to validate syntax and structure of the CSS.
  *
  */
-gulp.task('css:lint', () => gulp.src(`${pathBuilder.cssDistDir}/**/*.css`)
+gulp.task('css:lint', () => gulp.src([`${pathBuilder.cssDistDir}/**/*.css`])
```
Comment (DamianMullins) [inline tasks/css.js]: One tiny nitpick; remove the square braces :)

**Pull request merged**